Jenkinsfile

```diff
@@ -189,6 +147,29 @@ node('vetsgov-general-purpose') {
     }
   }
 
+  stage('Build: Redirects') {
+    if (shouldBail()) { return }
+
+    try {
+      def builds = [:]
+
+      for (int i=0; i<VETSGOV_BUILDTYPES.size(); i++) {
+        def envName = VETSGOV_BUILDTYPES.get(i)
+        builds[envName] = {
+          dockerImage.inside(args) {
+            sh "cd /application && npm --no-color run build:redirects -- --buildtype=${envName}"
+            sh "cd /application && echo \"${buildDetails('buildtype': envName, 'ref': ref)}\" > build/${envName}/BUILD.txt"
+          }
+        }
+      }
+
+      parallel builds
+    } catch (error) {
+      notify()
+      throw error
+    }
+  }
+
```

Comment:
I separated out the entry point for building Vets-to-VA redirects from the rest of the build script, because it's two completely different operations for two different sets of build-types.